packages/fccanalyses/package.py

```diff
@@ -126,6 +128,10 @@ def setup_run_environment(self, env):
             env.prepend_path("CPATH", join_path(awk_pydir, "include"))
             env.prepend_path("LD_LIBRARY_PATH", awk_pydir)
 
+        if self.spec.variants["onnx"].value:
+            env.prepend_path("LD_LIBRARY_PATH",
+                             self.spec["py-onnxruntime"].prefix.lib)
```

Comment:
```suggestion
                             self.libs.directories[0])
```
This is annoying but some distributions install to lib and others to lib64 so just adding lib won't work always. In this case it seems both Alma 9 and Centos 7 install the libraries to lib64 and not lib